assets/js/components/event/ParticipantsList.tsx

```diff
@@ -99,7 +99,8 @@ const Participant = ({ client, participant, eventChannel }: ParticipantProps) =>
     : "Participant";
 
   const isMyself: boolean = client.email == participant.email;
-  const isMyselfNotModeratorParticipant = !client.isModerator && role == "Participant" && isMyself;
+  const isMyselfNotModeratorParticipant =
```

Comment:
isMyselfParticipant implicate that im not a moderator. So isMyselfParticipant is better name I guess.